system/i18n/widgets/iframewidget.properties

```diff
@@ -0,0 +1,5 @@
+title=Iframe Widget
+description=Run HTML Code on the site
```

Comment:
Title should be "Iframe"
Description should be "Configure and embed an Iframe in your content"